src/App.js

```diff
@@ -1,9 +1,54 @@
 import './App.css';
-
+import Grid from '@mui/material/Grid';
+import Notes from './components/Notes';
+import { Container } from '@mui/material';
+import Heading from './components/Heading';
+import BasicModal from './components/Modal';
+import BottomNavigation from '@mui/material/BottomNavigation';
+import Paper from '@mui/material/Paper'
+import notesList from './data.json';
+import { LocationCity } from '@mui/icons-material';
 function App() {
   return (
-    <div className="App">
+    <div>
+    <Heading />
+    <Container sx={{marginTop: 5, marginBottom: 12, }}>
+    {notesList.map((notes) => (
+
+<Grid container spacing={5}> 
```

Comment:
each item when we iterate for the DOM should have a key. Check the https://reactjs.org/docs/lists-and-keys.html